.ci/atime/tests.R

```diff
@@ -66,61 +66,60 @@ pkg.edit.fun = function(old.Package, new.Package, sha, new.pkg.path) {
-#            For historical regressions, use 'Before', 'Regression', and 'Fixed' (otherwise something like 'Slow' or 'Fast' ideally).
+# - seconds.limit: The maximum median timing (in seconds) of an expression. No timings for larger N are computed past that threshold. Default of 0.01 seconds should be sufficient for most tests.
+# - Historical references (short version name = commit SHA1).
+#   For historical regressions, use version names 'Before', 'Regression', and 'Fixed'
+#   When there was no regression, use 'Slow' and 'Fast' 
 # @note Please check https://github.com/tdhock/atime/blob/main/vignettes/data.table.Rmd for more information.
 # nolint start: undesirable_operator_linter. ':::' needed+appropriate here.
-test.list <- list(
+test.list <- atime::atime_test_list(
   # Performance regression discussed in: https://github.com/Rdatatable/data.table/issues/4311
   # Fixed in: https://github.com/Rdatatable/data.table/pull/4440
-  "shallow regression fixed in #4440" = list(
-    pkg.edit.fun = pkg.edit.fun,
+  pkg.edit.fun = pkg.edit.fun,
+  "shallow regression fixed in #4440" = atime::atime_test(
     N = 10^seq(3,8),
-    setup = quote({
+    setup = {
       set.seed(1L)
       dt <- data.table(a = sample.int(N))
       setindexv(dt, "a")
-    }),
-    expr = quote(data.table:::shallow(dt)),
+    },
+    expr = data.table:::shallow(dt),
     # Before = "", This needs to be updated later as there are two issues here: A) The source of regression (or the particular commit that led to it) is not clear; B) Older versions of data.table are having problems when being installed in this manner (This includes commits from before March 20 2020, when the issue that discovered or first mentioned the regression was created)
     Regression = "b1b1832b0d2d4032b46477d9fe6efb15006664f4", # Parent of the first commit (https://github.com/Rdatatable/data.table/commit/0f0e7127b880df8459b0ed064dc841acd22f5b73) in the PR (https://github.com/Rdatatable/data.table/pull/4440/commits) that fixes the regression
     Fixed = "9d3b9202fddb980345025a4f6ac451ed26a423be"), # Merge commit in the PR that fixed the regression (https://github.com/Rdatatable/data.table/pull/4440)
 
   # Test based on: https://github.com/Rdatatable/data.table/issues/5424
   # Performance regression introduced from a commit in: https://github.com/Rdatatable/data.table/pull/4491
   # Fixed in: https://github.com/Rdatatable/data.table/pull/5463
-  "memrecycle regression fixed in #5463" = list(
-    pkg.edit.fun = pkg.edit.fun,
+  "memrecycle regression fixed in #5463" = atime::atime_test(
     N = 10^seq(3, 8),
-    setup = quote({
+    setup = {
       n <- N/100
       set.seed(2L)
       dt <- data.table(
         g = sample(seq_len(n), N, TRUE),
         x = runif(N),
         key = "g")
       dt_mod <- copy(dt)
-    }),
-    expr = quote(data.table:::`[.data.table`(dt_mod, , N := .N, by = g)),
```

Comment:
using atime_test, quote is no longer needed in setup, nor expr

Reply:
Sweet!